### DISCUSSION TITLE
Source fields are grayed out

### DISCUSSION THREAD

**Discussion by ghost** [atlasmap/atlasmap#3429] [Q&A] [answered]
Hello,

I imported the HL7 class files into source panel but I saw certain fields got grayed out and not expandable / mappable. 
I don't see any errors from log. 
Could you advise on how I can resolved it? I have a screen capture below.
Many thanks.

![image](https://user-images.githubusercontent.com/92883778/138321192-1bd46949-39fa-44cc-b19c-4cc4c09c97ee.png)

**Comment by igarashitm** [comment, marked as answer]
Those are unsupported fields, for example recursive one. You'd have to wait for https://github.com/atlasmap/atlasmap/issues/579 for recursive field support.
